site/content/docs/main/configuration.md

```diff
@@ -167,9 +167,11 @@ _This is Envoy's default setting value and is not explicitly configured by Conto
 
 The cluster configuration block can be used to configure various parameters for Envoy clusters.
 
-| Field Name        | Type   | Default | Description                                                                                                                                                             |
-| ----------------- | ------ | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| dns-lookup-family | string | auto    | This field specifies the dns-lookup-family to use for upstream requests to externalName type Kubernetes services from an HTTPProxy route. Values are: `auto`, `v4`, `v6`, `all` |
+| Field Name                        | Type   | Default | Description                                                                                                                                                                     |
+|-----------------------------------|--------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| dns-lookup-family                 | string | auto    | This field specifies the dns-lookup-family to use for upstream requests to externalName type Kubernetes services from an HTTPProxy route. Values are: `auto`, `v4`, `v6`, `all` |
+| max-requests-per-connection       | int    | none*   | This field specifies the maximum requests for upstream connections. If not specified, there is no limit                                                                         |
+| per-connection-buffer-limit-bytes | int    | none*   | This field specifies the soft limit on size of the cluster’s new connection read and write buffer. If not specified, Envoy defaults of 1MiB apply                               |
```

Comment:
I think the idea with the `*` next to defaults was to indicate defaults that _Envoy_ applied, rather than Contour -- so in this case, `per-connection-buffer-limit-bytes` should list a default of `1MiB` with a `*`, and then `max-requests-per-connection` does not need a `*` since there's no default.

If we're going to use that, please copy the footnote from `Timeout Configuration` to this section as well. It also looks like the leading `*` in that footnote was dropped at some point - please add it back so it's clear what that note refers to.